package.json

```diff
@@ -157,7 +157,7 @@
     "node-polyglot": "^2.4.0",
     "os-browserify": "^0.3.0",
     "p-debounce": "^4.0.0",
-    "p-memoize": "^7.1.1",
+    "p-memoize": "^4.0.1",
```

Comment:
Needed because of jest CJS shenanigans because later versions of `p-memoize` are now pure ESM modules, see https://github.com/sindresorhus/p-memoize/issues/29

Reply:
Update, since this only reugs tests and we don't care about the memoized functionality there, mocked ESM globally and reverted back the package.json entry to latest in dd9d60139a7728de87c1ff7ae2100ab481463364